src/behavior-not-considered-unsafe.md

```diff
@@ -37,3 +37,18 @@ See [RFC 560] for error conditions, rationale, and more details about
 integer overflow.
 
 [RFC 560]: https://github.com/rust-lang/rfcs/blob/master/text/0560-integer-overflow.md
+
+##### Logic errors
+
+Some safe functionality may impose extra logic constraints that cannot be
```

Comment:
```suggestion
Some safe functionality may impose extra logical constraints that can be
```

The way you have it written now, it's a double negative.

Reply:
Ah, yeah, that is my Spanish speaking :-)